src/pages/verify/getting-started/oidc-intro.mdx

```diff
@@ -200,6 +200,17 @@ Your next steps will depend on the [OpenID Connect flow type](/verify/getting-st
 
 After completing the steps specific to your chosen flow, you can proceed to [validating the JWT](/verify/getting-started/oidc-intro/#validate-the-jwt) received from successful authentication.
 
+<Highlight icon="info" warning>
+
+A more secure alternative to traditional authorization requests is [Pushed Authorization Requests (PAR)](/verify/guides/pushed-authorization-requests).
+
+PAR lets you initiate the authentication flow by sending [authorization request parameters](/verify/getting-started/oidc-intro/#authorize-request-parameters) directly to the authorization server via a POST request.
+This approach keeps parameters out of URLs, making requests more secure and protecting their integrity.
```

Comment:
Also, it protects their confidentiality if you are e.g. using PII in a login hint.